### PR TITLE
Credential services, workspace and notification related stuffs

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

**Comment by horadla23** [comment]
Is it planned to bring workspace permissions back? I think we should rely on UMS regarding permissions.

**Comment by topolyai5** [comment]
> Is it planned to bring workspace permissions back? I think we should rely on UMS regarding permissions.

We don't bring out the workspace functions, we've removed from the URL only. We don't know the life of the workspace in the future. It's easier than remove it and write it again.

File: cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/credential/CredentialPrerequisitesResult.java [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformRequest;
 import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult;
-import com.sequenceiq.cloudbreak.cloud.response.CredentialPrerequisitesV4Response;
+import com.sequenceiq.cloudbreak.cloud.response.CredentialPrerequisitesResponse;
 
 public class CredentialPrerequisitesResult extends CloudPlatformResult<CloudPlatformRequest<?>> {
 
-    private CredentialPrerequisitesV4Response credentialPrerequisitesV4Response;
+    private CredentialPrerequisitesResponse credentialPrerequisitesV4Response;
```
Comment (afarsang) [inline cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/credential/CredentialPrerequisitesResult.java]: would be nice without v4 the field name

File: workspace-authorization/build.gradle [hunk 2]
```diff
@@ -31,6 +31,7 @@ dependencies {
   compile group: 'org.apache.commons',            name: 'commons-lang3',                  version: apacheCommonsLangVersion
   compile group: 'org.springframework.security',  name: 'spring-security-jwt',            version: '1.0.8.RELEASE'
   compile group: 'org.springframework.retry',     name: 'spring-retry',                   version: '1.2.0.RELEASE'
+  compile group: 'io.swagger', name: 'swagger-jersey2-jaxrs', version: "1.5.21"
```
Comment (afarsang) [inline workspace-authorization/build.gradle]: we have property for swagger version in gradle.properties (swaggerVersion=1.5.21
)

**Pull request merged**